### PR TITLE
Updated readme with wabout feature

### PR DESCRIPTION
**Pull request by Rits1272**
## Issue that this pull request solves

Closes: #242 

## Proposed changes

Added wabout feature in readme

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Rits1272** [comment]
Please check @rpotter12 @xandao6

File: README.md [hunk 1]
```diff
@@ -77,7 +77,10 @@ python3 -m wplay -pull "AnyName"
 ```shell
 python3 -m wplay -schedule "AnyName"
 ```
-
+9. WAbout - Change your about section every second or every 30 min
```
Comment (rpotter12) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5225152)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5225152&bid=17130830)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5225152&bid=17130830)

File: README.md [hunk 2]
```diff
@@ -77,7 +77,10 @@ python3 -m wplay -pull "AnyName"
 ```shell
 python3 -m wplay -schedule "AnyName"
 ```
-
+9. WAbout - Change your about in every second or every 30 min.
```
Comment (rpotter12) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5225152)

**Pull request merged**